juju/juju
# Add scope to trust CLI for k8s models

Add a scope option to the trust command - when trusting charms on k8s models, the user is required to use `--scope=cluster` to confirm that the charm will be granted access to the entire cluster.
This is a stop gap until the permissions model is refined.

## QA steps

deploy a k8s charm
```
$ juju trust snappass-test
ERROR 'juju trust' currently grants full access to the cluster itself.
Set the scope to 'cluster' using '--scope=cluster' to confirm this choice.

$ juju trust snappass-test --scope=clu
ERROR scope "clu" not valid

$ juju trust snappass-test --scope=cluster
$

$ juju trust snappass-test --remove
$
```
